### PR TITLE
Added plane

### PR DESCRIPTION
**Pull request by anikan**
I found this cool plane model. What do you think.

### CODE REVIEW

File: Assets/World War II - American Plane/Materials/World War II - Propeller.mat [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  m_ShaderKeywords: 
+  m_LightmapFlags: 5
+  m_CustomRenderQueue: -1
+  stringTagMap: {}
+  m_SavedProperties:
+    serializedVersion: 2
+    m_TexEnvs:
+    - first:
+        name: _MainTex
+      second:
```
Comment (anikan) [inline Assets/World War II - American Plane/Materials/World War II - Propeller.mat:20]: What is this

File: Assets/World War II - American Plane/Materials/World War II - Propeller.mat [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  m_Shader: {fileID: 7, guid: 0000000000000000f000000000000000, type: 0}
+  m_ShaderKeywords: 
+  m_LightmapFlags: 5
+  m_CustomRenderQueue: -1
+  stringTagMap: {}
+  m_SavedProperties:
+    serializedVersion: 2
+    m_TexEnvs:
+    - first:
+        name: _MainTex
```
Comment (anikan) [inline Assets/World War II - American Plane/Materials/World War II - Propeller.mat:19]: redo this

Comment (anikan) [reply Assets/World War II - American Plane/Materials/World War II - Propeller.mat:19]: gud job

**Comment by connorasmith** [comment]
So proud

**Comment by connorasmith** [comment]
Out of context (from what I assume is a tutorial) this pull request is hilarious

**Pull request merged**